### PR TITLE
Ec2 provision

### PR DESCRIPTION
**Pull request by tyler-wenzel**
Provisioned ec2 instance with necessary security group. The ec2 instance is equipped with a startup script that spins up the customer full stack application and deploys it on a docker container. Appropriate security group configurations have been provided to allow for desired https and ssh traffic.

### CODE REVIEW

File: ec2.tf [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    sudo docker compose --env-file .env up -d
+
+  EOF
+
+  tags = {
+    Name = "loaf_application"
+  }
+}
+
+/* resource "aws_acm_certificate" "cert" {
```
Comment (brandoncorey) [inline ec2.tf]: Can you remove the commented out code before we merge?

**Review by brandoncorey — CHANGES_REQUESTED** [review]
Good work Tyler. Looks good, just review the requested change before merging.

File: ec2.tf (hunk 1 above)
Comment (tyler-wenzel) [reply ec2.tf]: The above code has been deleted out of the file.

**Review by brandoncorey — APPROVED** [review]
Looks good Tyler. Good job

**Pull request merged**